openmetadata-ui/src/main/resources/ui/src/components/common/success-screen/SuccessScreen.tsx

```diff
@@ -126,14 +125,11 @@ const SuccessScreen = ({
             {!isUndefined(fetchAirflowStatus) && (
               <div className="tw-flex-none">
                 <Button
-                  className={classNames('tw-self-center tw-py-1 tw-px-1.5', {
-                    'tw-opacity-40': isFetchingStatus,
-                  })}
+                  ghost
                   data-testid="airflow-status-check"
                   disabled={isFetchingStatus}
```

Comment:
```suggestion
                  loading={isFetchingStatus}
```

https://ant.design/~demos/components-button-demo-loading